package.json

```diff
@@ -27,14 +27,14 @@
   "license": "Apache-2",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/uport-project/uport-identity.git"
+    "url": "git+https://github.com/raininja/uport-identity.git"
```

Comment:
Please change this back to `uport-project`

Reply:
the reason I changed it was that uport-identity was a problem, still pullling in the soidity-sha3 package which caused it not to build.  I've since noticed that the tests fail on   MetaIdentityManagerSigner signRawTx signs transaction when using the web3.utils.soliditySha3 function.